release_notes/ocp-4-17-release-notes.adoc

```diff
@@ -1011,15 +1011,15 @@ BMER was deprecated in {product-title} version 4.15 and 4.16. With this release,
 [id="ocp-4-17-image-registry-bug-fixes_{context}"]
 ==== Image Registry
 
-* In {product-title} 4.14, installing a cluster with Azure AD Workload Identity was made generally available. With that feature, administrators can configure a Microsoft Azure cluster to use Azure AD Workload Identity. With Azure AD Workload Identity, cluster components use temporary security credentials that are managed outside of the cluster.
+* In {product-title} 4.14, installing a cluster with Microsoft Entra Workload ID was made generally available. With that feature, administrators can configure a Microsoft Azure cluster to use Microsoft Entra Workload ID. With Microsoft Entra Workload ID, cluster components use temporary security credentials that are managed outside of the cluster.
```

Comment:
"In {product-title} 4.14, installing a cluster with Microsoft Entra Workload ID was made generally available. With that feature, administrators can configure a Microsoft Azure cluster to use Microsoft Entra Workload ID. With Microsoft Entra Workload ID, cluster components use temporary security credentials that are managed outside of the cluster."

The use of the word "that" throws me a little. I think  "With this feature..." or "Using this feature..." might work better.